### ISSUE TITLE
COUNT() returns invalid results

### ISSUE THREAD

**Issue by diversario** [dbcli/mycli#233]
Using mycli:

```
mysql user@host:dbname> SELECT COUNT(id) FROM dbname.tablename WHERE fieldname = "value";
+---------------+
|   row_count() |
|---------------|
|            -1 |
+---------------+
Query OK, 2499 rows affected
Time: 0.188s
```

Using mysql CLI:

```
[host dbname]> SELECT COUNT(id) FROM dbname.tablename WHERE fieldname = "value";
+-----------+
| COUNT(id) |
+-----------+
|      2499 |
+-----------+
1 row in set (0.12 sec)
```

**Comment by amjith** [comment]
That is awful. Thanks for reporting it. I'll take a look.

**Comment by tsroten** [comment]
@diversario Does this happen on all tables or a particular one?

**Comment by diversario** [comment]
I think this happened on two tables but I'm not 100% sure.

**Comment by tsroten** [comment]
@diversario This is definitely odd behavior.

It would be helpful if you could set the mycli logging level to `debug` and post the relevant portions of the log.

Could you edit your `~/.myclirc` to have `log_level = debug` under the `[main]` section? Also, either set or not what `log_file` is set to. That should make the log more verbose.

**Comment by tsroten** [comment]
@diversario I submitted a related pull request about a week ago. Can you try the latest version of mycli from Github and see if the problem persists? Thanks!

**Comment by amjith** [comment]
You can install directly from master using this command: 

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by amjith** [comment]
Fixed as part of 1.6.0. Please upgrade: 

```
pip install -U mycli 
```
